content/applications/general/integrations/translate.rst

```diff
@@ -0,0 +1,122 @@
+user's text from another language, click the :guilabel:`...(three dot)` icon menu to the right of
+the chatter. Then, select :guilabel:`Translate`. The content translates to the *language* set on the
+user's preferences.
+
+.. seealso::
+   :ref:`language/change-user-language`
+
+.. image:: translate/google-translate.png
+   :align: center
+   :alt: Google Translate present in an Odoo database's chatter.
+
+Integration is simple with some configuration on *Google's Developer Console*, along with copying
+and pasting an *API key* into Odoo.
+
+Google API console
+==================
+
+A majority of the setup for integrating *Google Translate* into Odoo is done with *Google*. Once the
+following processes are complete an :guilabel:`API key` is created to input in Odoo.
+
+.. seealso::
+   `Google Translate setup on Google <https://cloud.google.com/translate/docs/setup>`_
+
+Create a new project
+--------------------
+
+To get started, go to the `Google API Console <https://console.developers.google.com>`_. Then, log
+in with a *Google Workspace* account, if there is one. If not, log in with a personal Gmail account
+(this should match the email address that has billing attached to it).
+
+Next, click :guilabel:`Create Project` on the far-right of the :guilabel:`OAuth consent screen`.
+
+.. tip::
+   If the *Google Developer Console* has existing projects, click the drop-down menu next to the
+   :guilabel:`Google Cloud` icon, and a pop-over window appears. Next, click :guilabel:`New Project`
+   top-right of the pop-over window.
+
+On the :menuselection:`New Project` screen, rename the :guilabel:`Project name` to `Odoo Translate`,
+and browse for the :guilabel:`Location`. Set the :guilabel:`Location` as the *Google Workspace
+organization*. If a personal Gmail account is being used, leave the :guilabel:`Location` as
+:guilabel:`No Organization`.
+
+.. image:: translate/new-project.png
+   :align: center
+   :alt: Project Name and Location for Google OAuth.
+
+Click on :guilabel:`Create` to finish this step.
+
+API Library
+-----------
+
+Next, the :guilabel:`Cloud Translation API` needs to be installed on this newly-created project.
+To do that, click :guilabel:`Library` in the left menu. Then, search the term `Cloud Translation
+API`, and click into the result. This should be a *Google Enterprise API* labeled `Cloud Translation
+API`.
+
+Click :guilabel:`Enable` to install the library on this project.
+
+.. important::
+   Using the *Google Translate* API **requires** a current billing account with `Google
+   <https://https://myaccount.google.com/>`_.
+
+Once a billing account is setup with *Google* and the library is enabled, click :guilabel:`Manage`
+to finish configuration on the API.
+
+Create Credentials
+------------------
+
+Now that the project is set up, and the library is enabled, credentials **must** be created. This
+includes the *API key*.
+
+To begin this process, click :guilabel:`Credentials` in the left sidebar menu.
```

Comment:
I believe a menuselection is a better fit here, since it is a menu the user is clicking on
```suggestion
To begin this process, click :menuselection:`Credentials` in the left sidebar menu.
```